api/database/assets/audit_mapper.py

```diff
@@ -6,5 +6,18 @@
     "email"                 : 4,
     "get_students"          : 5,
     "get_users"             : 6,
-    "get_student_forms"     : 7,
+    "get_student_info"      : 7,
+    'get_forms'             : 8,
+    'delete_blank_form'     : 9,
+    'update_form_name'      : 10,
+    'add_form'              : 11,
+    'get_blank_forms'       : 12,
+    'add_student'           : 13,
+    'get_form'              : 14,
+    'get_student_forms'     : 15,
+    'get_students_of_parent': 16,
+    'resend_forms'          : 17,
+    'get_all_forms'         : 18,
+    'submit_form'           : 19,
```

Comment:
We have this as numbers here as a key, but why not save it as a string? might be easier to parse in the future and do database migrations.